addons/actions.md

```diff
@@ -64,15 +64,26 @@ Note: The commandLine variable often has to use a special format where @@ needs
 
 ### HTTP Actions
 
-- `sendHttpGetRequest(String url)`: Sends an GET-HTTP request and returns the result as a String
+- `sendHttpGetRequest(String url)`: Sends a GET-HTTP request and returns the result as a String
+- `sendHttpGetRequest(String url, Map<String, String> headers, int timeout)`: Sends a GET-HTTP request with the given request headers, and timeout in ms, and returns the result as a String
 - `sendHttpPutRequest(String url)`: Sends a PUT-HTTP request and returns the result as a String
 - `sendHttpPutRequest(String url, String contentType, String content)`: Sends a PUT-HTTP request with the given content and returns the result as a String
+- `sendHttpPutRequest(String url, String contentType, String content, Map<String, String> headers, int timeout)`: Sends a PUT-HTTP request with the given content, request headers, and timeout in ms, and returns the result as a String
 - `sendHttpPostRequest(String url)`: Sends a POST-HTTP request and returns the result as a String
 - `sendHttpPostRequest(String url, String contentType, String content)`: Sends a POST-HTTP request with the given content and returns the result as a String
+- `sendHttpPostRequest(String url, String contentType, String content, Map<String, String> headers, int timeout)`: Sends a POST-HTTP request with the given content, request headers, and timeout in ms, and returns the result as a String
 - `sendHttpDeleteRequest(String url)`: Sends a DELETE-HTTP request and returns the result as a String
+- `sendHttpDeleteRequest(String url, Map<String, String> headers, int timeout)`: Sends a DELETE-HTTP request with the given request headers, and timeout in ms, and returns the result as a String
 
 Note: All HTTP Actions can have a last `timeout` parameter added in ms. eg. `sendHttpPostRequest(String url, String contentType, String content, int timeout)`
 
+For example:
+```javascript
+val headers = <String, String>newHashMap
```

Comment:
AFAIK we can simplify this and the following line to

```
val headers = newHashMap("Cache-control" -> "no-cache")
```

Reply:
Agreed! Tested and submitted an update.